src/appleseed/renderer/modeling/light/sunlight.cpp

```diff
@@ -428,6 +482,15 @@ namespace
                 21097.9f, 20728.3f, 20240.4f
             };
 
+            // Limb darkening.
```

Comment:
Is there a reference for this code? Paper name, or equation number?

Reply:
Add the paper, not sure if it's the best way to add it though.